packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -2052,6 +2055,9 @@ void main() {
       await tester.pumpWidget(buildEditableText());
 
       for (final SingleActivator activator in allModifierVariants(LogicalKeyboardKey.arrowDown)) {
+        if (activator.shift) {
+          continue;
+        }
```

Comment:
This test loops through a variety of common shortcut activators (shift, alt, meta, etc). I added this line to skip if the current iteration activator is shift, since we're now handling shift + arrow differently

Reply:
Ah ok, this is inside the `Web does not accept` group. That makes sense.

Could you please put this explanation in a comment?